src/views/accounts/AccountActivity/AccountActivityList.vue

```diff
@@ -71,7 +71,27 @@ export default {
         hasImport: false,
         hasDatePicker: true,
         searchConfig: {
-          exclude: ['resource_type']
+          exclude: ['resource_type', 'action'],
+          options: [{
+            value: 'action',
+            label: this.$t('Action'),
+            type: 'choice',
+            children: [
+              {
+                default: true,
+                value: 'view',
+                label: this.$t('View')
+              },
+              {
+                value: 'update',
+                label: this.$t('Update')
+              },
+              {
+                value: 'delete',
+                label: this.$t('Delete')
+              }
+            ]
+          }]
         }
       }
     }
```

Comment:
There doesn't appear to be any specific syntax or logical errors with the current code snippet. However, there may be a few areas that could benefit from review:

1. The `hasImport` property seems redundant since it's simply defined as false but then used later without reference.

To optimize readability and maintainability, remove unnecessary properties (e.g., removing `exclude` if no longer needed). 

Additionally, ensure consistent formatting and naming conventions throughout the file. For example, consider renaming variables (`$t`, `resource_type`, etc.) for better clarity in context. Adjustments will help improve the project quality significantly over time.